package.json

```diff
@@ -79,11 +80,11 @@
     "eslint-plugin-react-hooks": "^4.2.0",
     "i18n-js": "^3.8.0",
     "jest": "^26.6.3",
-    "jest-expo": "^40.0.2",
-    "prettier": "^2.2.1",
+    "jest-expo": "^42.0.0",
+    "prettier": "~2.2.1",
     "react-native-svg-transformer": "^0.14.3",
     "react-test-renderer": "^17.0.2",
-    "typescript": "~4.2.4"
+    "typescript": "~4.0.0"
```

Comment:
I noticed there's a downgrade here, how come ah?

Reply:
as discussed, will update `typescript` to `~4.2.4~